coalib/bearlib/languages/LanguageDefinition.py

```diff
@@ -7,7 +7,7 @@
 
 class LanguageDefinition(SectionCreatable):
 
-    def __init__(self, language: str):
+    def __init__(self, language: str, coalang_path=None):
```

Comment:
`coalang_path` -> `external_coalang_path`?

Also how about making the variable represent the *directory* in which the `language.coalang` file is located? That way we'll be consistent with the current design (with your current design, `language` is a dummy parameter if the user wants to use a custom coalang).